nylas/client/restful_models.py

```diff
@@ -20,7 +20,14 @@ def typed_dict_attr(items, attr_name=None):
     return dct
 
 
-class NylasAPIObject(dict):
+def _is_subclass(cls, parent):
+    for base in cls.__bases__:
+        if base.__name__.lower() == parent:
+            return True
+    return False
+
+
+class RestfulModel(dict):
```

Comment:
We do this because the Neural API series of endpoints don't support the same operations as the typical Nylas API objects. Thus this allows us to extend the minimum required functionality shared by both the typical Nylas API objects and Neural API objects.